ui/components-react/src/components-react/filter-builder/FilterBuilderRule.tsx

```diff
@@ -54,7 +54,12 @@ export function FilterBuilderRuleRenderer(props: FilterBuilderRuleRendererProps)
     return <FilterBuilderRuleValue property={prop} onChange={onRuleValueChange} value={value} />;
   }, [value, ruleValueRenderer, onRuleValueChange]);
 
-  return <div className="rule">
+  return <div className="rule" tabIndex={-1}>
```

Comment:
What function does this `tabIndex={-1}` serve? I removed this attribute and did not observe any differences in keyboard navigation on Firefox.